### DISCUSSION TITLE
Software too slow on Intel Core i5-10400

### DISCUSSION THREAD

**Discussion by TopoSan** [madMAx43v3r/chia-plotter#247] [Q&A]
Hello everybody,
I have run mad max on an Intel i5-10400 machine, with 16GB of RAM, and a 1TB SSD. The problem is that the software does not take advantage of the cores and the generation of the phases is very slow. I don't know if it has to do with the software or if it is a matter of my machine. I run the application with 10 threads. Thanks for the help.
![PlotsI5](https://user-images.githubusercontent.com/11886342/121686145-1a14c600-cac1-11eb-9bbf-5012fca68999.JPG)

**Comment by SebMoore** [comment]
Well given by the usage of your D: drive, you appear to be creating the plots onto a USB drive. Is this the case?

**Reply by TopoSan** [reply]
Yes.But it works two slow when i change it to SSD drive too...
![ssd](https://user-images.githubusercontent.com/11886342/121690922-b8575a80-cac6-11eb-8eb1-9f313e392c00.JPG)

**Reply by SebMoore** [reply]
Well you can clearly see that your SSD is at maximum usage. As such, the plotter is working perfectly: it cannot go any faster because your SSD cannot go any faster.

**Reply by TopoSan** [reply]
Thank you..but...how can i do to free my SSD when is plotting¿?It has 1Tb ...i configure with 2 tmp drives in SSD.Thank you!!
